### DISCUSSION TITLE
SQLiteStudio 3.4.10 Windows x64 Portable download link broken

### DISCUSSION THREAD

**Discussion by patevs** [pawelsalawa/sqlitestudio#5167] [General]
### Details

The link to download the Windows x64 Portable version of SQLiteStudio version 3.4.10 is broken and leads to a 404 Not Found.

### Steps to reproduce

1. Open link: https://github.com/pawelsalawa/sqlitestudio/releases/download/3.4.10/sqlitestudio_x64-3.4.10.zip
2. Observe 404 Not Found.

### SQLiteStudio version

 3.4.10 Windows x64 Portable

**Comment by includewins0ck2** [comment]
Yes,the same problem

**Comment by pawelsalawa** [comment]
Thanks for reporting this. The link is fixed now.
